pkg/controller/pcideviceclaim/pcideviceclaim_controller.go

```diff
@@ -331,49 +286,144 @@ func (h Handler) rebindAfterReboot() error {
 
 func (h Handler) reconcilePCIDeviceClaims(name string, pdc *v1beta1.PCIDeviceClaim) (*v1beta1.PCIDeviceClaim, error) {
 
-	if pdc == nil || pdc.DeletionTimestamp != nil {
+	if pdc == nil || pdc.DeletionTimestamp != nil || (pdc.Spec.NodeName != h.nodeName) {
```

Comment:
```suggestion
	if pdc.Spec.NodeName != h.nodeName {
```
dropped check to see if device passthrough is already enabled, as we still need to start the plugin if needed.